.github/workflows/ci.yml

```diff
@@ -59,15 +59,19 @@ jobs:
         packages_dir: dist-wheel
         verbose: true
 
-# Later: when tiledbsoma is released
-#    - name: Publish package to PyPI
-#      if: matrix.os == 'ubuntu-22.04' && github.event_name == 'release' && !github.event.release.prerelease
-#      uses: pypa/gh-action-pypi-publish@master
-#      with:
-#        user: __token__
-#        password: ${{ secrets.PYPI_TOKEN }}
-#        packages_dir: dist-wheel
-#        verbose: true
+    - name: Publish package to PyPI
+      # Until general release of the main branch of tiledbsoma we'll follow the convention of tagging releases
+      # 0.5.0a1, 0.5.1a1, etc -- always with the "a1" suffix -- that way PyPI will automagically make these
+      # "prereleases". Then:
+      # pip install tiledbsoma -> 0.1.12 (or whatever the main-old branch is at)
+      # pip install --pre tiledbsoma -> 0.5.0a1 (or whatever the main branch is at)
+      if: matrix.os == 'ubuntu-22.04' && github.event_name == 'release' && !github.event.release.prerelease
```

Comment:
The last part—whether you want to include `github.event.release.prerelease`—depends upon how you want to mark things in GitHub releases. I am inclined to *not* use it, so that you will still generate a PyPI release from a release that is marked as a pre-release on GitHub, like I did for [`v0.8.0b1` of `tiledb-cloud`](https://github.com/TileDB-Inc/TileDB-Cloud-Py/releases/tag/v0.8.0b1).

I might add a check to ensure we don’t accidentally release a "full" version before we’re ready:

`(contains(github.event.release.tag_name, "a") || contains(github.event.release.tag_name, "b"))`

to ensure that there's an `a` or `b` somewhere in the release name (i.e., you can't accidentally release `v0.5.0`, but you can release `v0.5.0b1`.

Reply:
@thetorpedodog 
https://github.com/single-cell-data/TileDB-SOMA/pull/366/commits/d8a639f19a502871f0a45c8771783f0476931871